Manual/.gitignore

```diff
@@ -1,11 +1,5 @@
 # Manual output and temporary files
 *.htm
 *.con
-ags.ref
-ags.chm
-ags.chw
-ags.hhc
-ags.hhk
-ags.hhp
-ags.htx
-ags-help.chm
+ags.*
```

Comment:
I don't think this is correct, perhaps better filter out by extensions rather than "ags" prefix.
Also ags-help.chm resurfaces because of this.